modules/tools/vault-less-secure/main.tf

```diff
@@ -61,7 +61,7 @@ resource "helm_release" "vault" {
       vault_hostname                = var.vault_hostname
       vault_service_account_arn     = var.vault_service_account_arn
       vault_credentials_secret_name = local.vault_secret_name
-      vault_config                  = indent(6, templatefile("${path.module}/vault-config.hcl.tpl", {
+      vault_config = indent(6, templatefile("${path.module}/vault-config.hcl.tpl", {
```

Comment:
let's run `terraform fmt` to undo some of these whitespace changes

Reply:
this was the result of `terraform fmt`